sdk/resource/resource_test.go

```diff
@@ -769,3 +770,30 @@ func TestWithContainer(t *testing.T) {
 		string(semconv.ContainerIDKey): fakeContainerID,
 	}, toMap(res))
 }
+
+func TestResourceRace(t *testing.T) {
```

Comment:
```suggestion
func TestResourceConcurrentSafe(t *testing.T) {
```

per https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#testing